src/satisfia/util/distribution.py

```diff
@@ -40,7 +38,7 @@ def var(self, *, precision=64):
 		return sum((samples - E) ** 2) / (precision - 1) # sample variance
 
 	def stddev(self, *, precision=64):
-		return torch.sqrt(self.var(precision))
+		return math.sqrt(self.var(precision))
```

Comment:
would you mind this? we don't need it yet, and it messes up with browser support

Reply:
ok for now, at some point we might however need it for training.
what about 
```
try:
  import torch
  sqrt = torch.sqrt
except:
  sqrt = math.sqrt  # or numpy?
```

Reply:
My guess would be that torch can be relegated to files that are imported only from the training part, which in turn are not imported by the web book, so no problem arises.

If that turns out not to be true, yes, your solution should work